src/assets/translations/fr/main.json

```
@@ -2477,7 +2499,58 @@
+      },
+      "why": {
+        "title": "Pourquoi miser vos FOX ?",
+        "body": "La mise de vos FOX vous permet de gagner régulièrement des paiements en RUNE provenant de la trésorerie de la DAO, fournissant un flux constant de revenus passifs en RUNE par époque. De plus, une partie des RUNE reçus par la DAO est utilisée pour acheter des FOX sur le marché libre et les brûler, réduisant ainsi la masse monétaire totale. La mise promeut également un alignement de ShapeShift DAO avec l'écosystème THORChain, contribuant ainsi à leur croissance et à leurs succès. Le processus de mise nécessite une période minimale de 28 jours avant la démise, encourageant l'engagement et la stabilité au sein de la communauté."
+      },
+      "stake": {
+        "title": "Comment miser mes FOX ?",
+        "body": "Pour miser vos FOX, vous utilisez l'interface de mise pour verrouiller vos jetons FOX dans le contrat de mise. Vous devrez signer un message ou une transaction pour désigner l'adresse où vos récompenses RUNE seront payées. Lorsque vous aurez misé, vous commencerez à gagner des récompenses RUNE par bloc dans le cadre de vos avantages de mise."
+      },
+      "unstake": {
+        "title": "Que se passe-t-il lorsque je démise ?",
+        "body": "Lorsque vous décidez de démiser, il y a une période minimale de 28 jours avant la démise. Pendant ce temps, vos FOX restent verrouillés et vous cessez de recevoir des émissions RUNE. Après la période de démise, vous pourrez débloquer et récupérer vos jetons FOX normaux."
+      },
+      "cooldown": {
+        "title": "Quelle est la durée de la période de repos ?",
+        "body": "La période de repos pour le retrait des jetons FOX est de 28 jours. Pendant ce temps, vos jetons restent verrouillés et ne peuvent être transférés."
+      },
+      "multipleUnstaking": {
+        "title": "Puis-je avoir plusieurs montants en cours de démise ?",
+        "body": "Oui, vous pouvez avoir plusieurs montants en cours de mise. Chaque action de démise aura sa propre période de repos de 28 jours."
+      },
+      "connect": {
+        "title": "Quel est le rapport entre le total de frais collectés, le total misé en FOX, le pool d'émissions et le montant de FOX brûlés ?",
+        "body": "Les frais collectés par la DAO sont en RUNE natifs. Une partie de ces RUNE est distribuée aux miseurs FOX en guise de récompenses, tandis qu'une autre partie est utilisée pour acheter des FOX sur le marché libre et les brûler, réduisant ainsi la masse monétaire globale de FOX. Le montant total misé en FOX influence la distribution des récompenses en RUNE, car les récompenses sont distribuées au prorata du montant des FOX misés. Le pool d'émissions représente la quantité de FOX disponibles pour les récompenses des mises, et le montant de FOX brûlés est la partie des FOX achetés et brûlés à l'aide des RUNE collectés grâce aux frais."
```

feels more natural to me but meh, both work perfectly fine! 
```suggestion
        "body": "Les frais collectés par la DAO sont en RUNE natif. Une partie de ces RUNE est distribuée aux miseurs FOX en guise de récompenses, tandis qu'une autre partie est utilisée pour acheter des FOX sur le marché libre et les brûler, réduisant ainsi la masse monétaire globale de FOX. Le montant total misé en FOX influence la distribution des récompenses en RUNE, car les récompenses sont distribuées au prorata du montant des FOX misés. Le pool d'émissions représente la quantité de FOX disponibles pour les récompenses des mises, et le montant de FOX brûlés est la partie des FOX achetés et brûlés à l'aide des RUNE collectés grâce aux frais."
```

---

As I've used plurals for most of the other symbols/amounts in this string and others explanations I think that the plural works better. Thanks for the suggestion though!